Library/Formula/compcert.rb

```diff
@@ -28,6 +47,8 @@ def install
     system "./configure", "-prefix", prefix, "ia32-macosx"
     system "make", "all"
     system "make", "install"
+
+    rm_r coq_prefix
```

Comment:
OK, I may just kill the coq_prefix local variable then since it's only used twice.